src/audio/effects.c

```diff
@@ -370,12 +370,12 @@ s32 adsr_update(struct AdsrState *adsr) {
             adsr->currentHiRes = (adsr->current << 0x10);
 #endif
             adsr->state = ADSR_STATE_LOOP;
+            // fallthrough
 
+        case ADSR_STATE_LOOP:
 #ifdef VERSION_SH
             restart:
 #endif
```

Comment:
this seems to change shindou functionality 🤔

Reply:
Yeah, this would be a pretty noticeable bug. Please fix.

Reply:
I reverted it and added another `// fallthrough` instead to fix the warning.